base/unicode/utf8.jl

```diff
@@ -101,26 +101,33 @@ sizeof(s::UTF8String) = sizeof(s.data)
 
 lastidx(s::UTF8String) = length(s.data)
 
+isfirstbyte(c::UInt8) = (c & 0xc0) != 0x80 # == !is_valid_continuation(c)
```

Comment:
I wasn't sure if it would be optimised to the same code, thanks for the clarification. For me it also aids readability, but I'm happy to remove.